examples/miscellaneous/miscellaneous_ex2/miscellaneous_ex2.C

```diff
@@ -436,7 +441,7 @@ void assemble_helmholtz(EquationSystems & es,
       // If the element has no neighbor on a side then that
       // side MUST live on a boundary of the domain.
       for (auto side : elem->side_index_range())
-        if (elem->neighbor(side) == libmesh_nullptr)
+        if (elem->neighbor_ptr(side) == libmesh_nullptr)
```

Comment:
Ha!  I thought I finally got rid of the last of our deprecated code when I built with the new --disable-deprecated, but I never tried --enable-complex --disable-deprecated.  Thanks!